src/Core/Checkout/Cart/Order/OrderConverter.php

```diff
@@ -143,6 +143,21 @@ public function convertToOrder(Cart $cart, SalesChannelContext $context, OrderCo
                 $context->getContext(),
                 $shippingAddresses
             );
+
+            // In order to reference the primary order delivery we need to set ids. The primary order delivery is the
+            // order delivery with the highest shipping costs (i.e. _not_ a shipping discount).
+            if ($cart->getDeliveries()->count() > 0) {
+                $primaryOrderDeliveryId = Uuid::randomHex();
+                usort(
+                    $data['deliveries'],
+                    function (array $deliveryA, array $deliveryB) {
+                        // @phpstan-ignore-next-line
```

Comment:
sorry, if I jump in here, also it is still a draft :see_no_evil: 

but I cannot ignore annotations like this :grin: if you really cannot fix the error, please use `@phpstan-ignore` with the proper error identifier and a comment why this error could not be fixed. also see https://phpstan.org/user-guide/ignoring-errors